src/blockchain/utils.ts

```diff
@@ -10,3 +10,13 @@ export function amountToWei(amount: BigNumber, token: string): BigNumber {
   const precision = tokens[token].precision;
   return amount.times(new BigNumber(10).pow(precision));
 }
+
+export const addressToBytes32 = (x: string, prefix = true) => {
```

Comment:
we need tests for these 2

Reply:
No, we dont! This shouldn't be there at all :)

Reply:
even better then! 😆